src/mito2/src/config.rs

```diff
@@ -147,10 +147,10 @@ impl Default for MitoConfig {
             vector_cache_size: ReadableSize::mb(512),
             page_cache_size: ReadableSize::mb(512),
             selector_result_cache_size: ReadableSize::mb(512),
-            enable_experimental_write_cache: false,
-            experimental_write_cache_path: String::new(),
-            experimental_write_cache_size: ReadableSize::gb(5),
-            experimental_write_cache_ttl: None,
+            enable_write_cache: false,
+            write_cache_path: String::new(),
+            write_cache_size: ReadableSize::gb(10),
```

Comment:
We should still set the default cache size to 5GB.

Now, the total cache size of the write cache and read cache is 10GB, and each cache uses half of this size.

Reply:
fixed, happy new year